src/lib/containers/CardContainerLogic.tsx

```diff
@@ -1,32 +1,42 @@
+import { cloneDeep, isEqual } from 'lodash-es'
 import * as React from 'react'
 import { SynapseClient, SynapseConstants } from '../utils'
 import { getNextPageOfData } from '../utils/functions/queryUtils'
 import {
   insertConditionsFromSearchParams,
   KeyValue,
-  SQLOperator,
   parseEntityIdFromSqlStatement,
+  SQLOperator,
 } from '../utils/functions/sqlFunctions'
+import { DEFAULT_PAGE_SIZE } from '../utils/SynapseConstants'
 import { QueryBundleRequest, QueryResultBundle } from '../utils/synapseTypes/'
 import CardContainer from './CardContainer'
 import { GenericCardSchema, IconOptions } from './GenericCard'
+import { IconSvgOptions } from './IconSvg'
 
 /**
  * TODO: SWC-5612 - Replace token prop with SynapseContext.accessToken
- * 
+ *
  * This wasn't done because Enzyme's shallow renderer is not currently
  * compatible with the `contextType` field in the React 16+ context API.
- * 
+ *
  * This can be fixed by rewriting tests to not rely on the shallow renderer.
- * 
+ *
  * See here: https://github.com/enzymejs/enzyme/issues/1553
  */
 
-// TODO: this import nearly doubles the package size of SRC as a UMD build by ~400KB
-// will have to find a way to use individual lodash packages instead of the entire thing
```

Comment:
I don't think this is still true--we're importing individual exports from lodash-es and rollup is doing its job w/ tree shaking

Reply:
Also...it would be great if 400KB was still "nearly doubling" 😂